container-app.tf

```diff
@@ -156,6 +156,15 @@ resource "azapi_resource" "default" {
     }
   })
 
+  dynamic "identity" {
+    for_each = local.container_app_identities != null ? toset([1]) : toset([])
```

Comment:
If `local.container_app_identities` default value is `{}` then you can lose the condition here and just leave it as `for_each = local.container_app_identities`

Reply:
Good shout, amended!